### PR TITLE
landingpage changes and assets folder changes

### PR DESCRIPTION
**Pull request by jaindra-bu**

### CODE REVIEW

File: code/packages/web/src/components/JobCard.tsx
Comment (tushar-kendre) [inline code/packages/web/src/components/JobCard.tsx:1]: No inline styles for any tags

File: code/packages/web/src/pages/dashboard/dashboard_pages/applicant_job_listings/useJobListings.ts
Comment (tushar-kendre) [inline code/packages/web/src/pages/dashboard/dashboard_pages/applicant_job_listings/useJobListings.ts:1]: Dummy data?

**Review by tushar-kendre — CHANGES_REQUESTED** [review]
Please see the comments on the files

File: code/packages/web/src/components/JobCard.tsx
Comment (jaindra-bu) [reply code/packages/web/src/components/JobCard.tsx:1]: done, changed

File: code/packages/web/src/pages/dashboard/dashboard_pages/applicant_job_listings/useJobListings.ts
Comment (jaindra-bu) [reply code/packages/web/src/pages/dashboard/dashboard_pages/applicant_job_listings/useJobListings.ts:1]: yes

**Review by tushar-kendre — APPROVED** [review]

**Review by hemant1026 — APPROVED** [review]

**Pull request merged**